src/lib/dbMaintenance.js

```diff
@@ -0,0 +1,7 @@
+import { QueryTypes } from 'sequelize';
+import { sequelize } from '../models';
+
+export default async function dbMaintenance() {
+  await sequelize.query('VACUUM FULL;', { type: QueryTypes.RAW });
+  await sequelize.query('REINDEX DATABASE ttasmarthub;', { type: QueryTypes.RAW });
```

Comment:
Can we get the name of the db dynamically here without exposing it, since it won't be ttasmarthub in deployed envs?

Reply:
I will look into that

Reply:
do you have an example where we do that already?